Sources/Compiler/IR/Module.swift

```diff
@@ -39,6 +39,24 @@ public struct Module {
   /// The module's name.
   public var name: String { program.ast[decl].name }
 
+  /// Accesses the function at given index.
+  public subscript(function index: Functions.Index) -> Function {
+    _read   { yield  functions[index] }
+    _modify { yield &functions[index] }
+  }
```

Comment:
```suggestion
  /// Accesses the function at `i`.
  public subscript(_ i: Functions.Index) -> Function {
    _read { yield functions[i] }
    _modify { yield &functions[i] }
  }
```

or even, since function indices _are_ stable,

```suggestion
  /// Accesses the given function.
  public subscript(_ f: Functions.Index) -> Function {
    _read { yield functions[f] }
    _modify { yield &functions[f] }
  }
```

Reply:
As mentioned earlier, I want to keep the label.